ui/src/app/pages/admin/admin-user-common.tsx

```diff
@@ -221,6 +221,30 @@ export const displayTierBadgeByRequiredModule = (
   );
 };
 
+export const getEraNote = (profile: Profile): string => {
+  const requiredForRT = isEraRequiredForTier(
+    profile,
+    AccessTierShortNames.Registered
+  );
+  const requiredForCT = isEraRequiredForTier(
+    profile,
+    AccessTierShortNames.Controlled
+  );
+  let note = `*eRA Commons requirements vary by institution. This user's institution 
```

Comment:
please add a space after the *

also: please add a * after the name "ERA Commons" in the table

Reply:
Done.

Reply:
Do you mean to update note to `* eRA Commons *` in second comment? I added `*` at the end of string.

Reply:
I mean to add it in the left hand column of the table.  The column with all the module names.

Reply:
I see. The module names are read from `getAccessModuleConfig.adminPageTitle`. We should not modify `adminPageTitle` because it's used in other places. I could copy values of `adminPageTitle` but that would create duplicated code. Do you have a better way to do this?